tests/e2e/main.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+from botocore.exceptions import ClientError
+import boto3
+import time
+import requests
+
+# Create AWS CloudWatch client
+aws_client = boto3.client('logs')
+
+def generate_log_event(log_group_name,log_stream_name):
+    """
+    Upload a log event to the specified AWS CloudWatch Logs log group
+    """
+
+    try:
+        log_event = aws_client.put_log_events(
+            logGroupName  = log_group_name,
+            logStreamName = log_stream_name,
+            logEvents=[
+                {
+                    'timestamp': epoch_timestamp_in_ms,
+                    'message': message_content
+                },
+            ],
+        )
+        print("Generated log event: %s" % log_event)
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'DataAlreadyAcceptedException' or e.response['Error']['Code'] == 'InvalidSequenceTokenException':
```

Comment:
Is it an expected situation? The documentation says:
```
An upload in a newly created log stream does not require a sequence token
```
https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html

and here we have a new log stream, right?

Reply:
Ahh, just realized -- this test can be run outisde of Travis on an existing log stream. Is this the reason to include this logic?

Reply:
Exactly:)